lib/signaling/v2/peerconnection.js

```diff
@@ -1036,11 +1066,12 @@ class PeerConnectionV2 extends StateMachine {
       });
     }
     return this._peerConnection.setLocalDescription(description).catch(error => {
-      this._log.warn(`Calling setLocalDescription with an RTCSessionDescription of type "${description.type}" failed with the error "${error.message}".`);
-      if (description.sdp) {
-        this._log.warn(`The SDP was ${description.sdp}`);
-      }
-      throw new MediaClientLocalDescFailedError();
+      const message = `Calling setLocalDescription with an RTCSessionDescription of type "${description.type}" failed`;
+      this._log.warn(message, { sdp: description.sdp, error });
```

Comment:
@makarandp0 Please retain the original way of printing the message. Also, we need to check if sdp exists since a rollback session description has sdp as null.
```suggestion
this._log.warn(`Calling setLocalDescription with an RTCSessionDescription of type "${description.type}" failed with the error "${error.message}".`);
      if (description.sdp) {
        this._log.warn(`The SDP was ${description.sdp}`);
      }
```